File: Iniciante/1101 - Sequência de Números e Soma/TypeScript/1101.ts
import * as fs from "fs";

function main(): void {
  const input: number[] = fs.readFileSync(0, "utf8").trim().split(/\s+/).map(Number);

  let index: number = 0;

  while (true) {
    let m: number = input[index++];
    let n: number = input[index++];

    if (m <= 0 || n <= 0) {
      break;
    }

    let sum: number = 0;
    let start: number = Math.min(m, n);
    let end: number = Math.max(m, n);

    let line: string = "";

    for (let i = start; i <= end; i++) {
      line += i + " ";
      sum += i;
    }

    console.log(line + "Sum=" + sum);
  }
}

main();

File: Iniciante/1101 - Sequência de Números e Soma/Python/1101.py
def main():
    while True:
        m, n = map(int, input().split())

        if m <= 0 or n <= 0:
            break

        total_sum = 0
        start = min(m, n)
        end = max(m, n)

        for i in range(start, end + 1):
            print(i, end=" ")
            total_sum += i

        print(f"Sum={total_sum}")


if __name__ == "__main__":
    main()
